website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -13,6 +13,16 @@ description: This guide walks through an example of building a simple memcached-
 - Access to a Kubernetes v1.11.3+ cluster (v1.16.0+ if using `apiextensions.k8s.io/v1` CRDs).
 - User logged with admin permission. See [how to grant yourself cluster-admin privileges or be logged in as admin][role-based-access-control]
 
+## Overview
+
+By using operators, it’s possible not only to provide all expected resources but also to manage them dynamically, programmatically, and at execution time. To illustrate this idea, imagine if someone accidentally changed a configuration or removed a resource by mistake; in this case, the operator could fix it without any human intervention. 
+
+We will create a sample project to let you know how it works and this sample will:
+
+- Create a memcached Deployment if it doesn't exist
```

Comment:
It seems this is describing what the controller will do? I think it might be better to do a list of all the topics covered in the tutorial. something like
- create a new API by generating a CRD
- create a controller that will watch that CRD type and xyz (bullets you wrote here explaining reconciliation)
- install the operator to your cluster by deploying those resources
- use the operator by creating instances of your new type

Reply:
The idea here is to clarify what are the goals that will be achieved with the sample that is built in this tutorial.
So, the operations done by the operator is implemented in the controller. Then, consequently, I agree that it shows what the controller does since it is what the operator does.